client/rest-high-level/src/main/java/org/elasticsearch/client/SecurityRequestConverters.java

```diff
@@ -153,17 +158,19 @@ static Request hasPrivileges(HasPrivilegesRequest hasPrivilegesRequest) throws I
     static Request clearRealmCache(ClearRealmCacheRequest clearRealmCacheRequest) {
         RequestConverters.EndpointBuilder builder = new RequestConverters.EndpointBuilder()
             .addPathPartAsIs("_security/realm");
-        if (clearRealmCacheRequest.getRealms().isEmpty() == false) {
+        if (!clearRealmCacheRequest.getRealms().isEmpty()) {
             builder.addCommaSeparatedPathParts(clearRealmCacheRequest.getRealms().toArray(Strings.EMPTY_ARRAY));
         } else {
             builder.addPathPart("_all");
         }
         final String endpoint = builder.addPathPartAsIs("_clear_cache").build();
         Request request = new Request(HttpPost.METHOD_NAME, endpoint);
-        if (clearRealmCacheRequest.getUsernames().isEmpty() == false) {
-            RequestConverters.Params params = new RequestConverters.Params(request);
+        if (!clearRealmCacheRequest.getUsernames().isEmpty()) {
```

Comment:
can you please revert this change and all the similar ones? we tend to prefer the `boolean == false` over `!boolean` as the exclamation mark can easily be missed.

Reply:
can you please revert this?